decidim-assemblies/spec/shared/manage_assemblies_examples.rb

```diff
@@ -152,7 +152,7 @@
       uncheck :assembly_scopes_enabled
 
       expect(page).to have_selector("#assembly_scope_id.disabled")
-      expect(page).to have_selector("#assembly_scope_id .picker-values div input[disabled]", visible: false)
+      expect(page).to have_selector("#assembly_scope_id .picker-values div input[disabled]", visible: :visible)
```

Comment:
Should we use `:all` as stated by @ahukkanen ?

Reply:
Yes, you're right! This error is due to a confusion in rubocop documentation I linked on the PR description.